lib/src/base-styles/_nx-forms.scss

```diff
@@ -10,88 +10,74 @@
 
 .nx-form {
   @include container-vertical;
-  border-top: $nx-border-dark;
   margin: 0;
-  padding-top: 20px;
   position: relative; // to make a containing block for validation popovers
 }
 
-.nx-form--simple {
-  border-top: none;
-  padding: 0;
-}
+.nx-form-footer {
+  @include container-horizontal;
 
-// for a form that is the entirety of a modal
-.nx-form--modal-form {
-  padding-top: 0;
+  border-top: 1px solid #e9e9e9;
+  padding: $nx-spacing-xl $nx-spacing-l;
+  text-align: right;
```

Comment:
I thought we decided that the form footer should include an `.nx-btn-bar` and that `.nx-btn-bar` should get this text-align style.

Reply:
> I thought we decided that the form footer should include an .nx-btn-bar and that .nx-btn-bar should get this text-align style.

Whatever we decide here may need to consider what the form footer is supposed to look like in case of a submit error.  @dturner32 ?

Reply:
correct, nx-btn bar should be included in the footer of a form . As for the error message handling, similar to the mock up of the design for the modal view, the error would appear above the buttons. I will create a mock up for that and post to zeplin

Reply:
> As for the error message handling, similar to the mock up of the design for the modal view, the error would appear above the buttons. I will create a mock up for that and post to zeplin

Keep in mind that if both Retry and Submit are visible at the same time, they'd presumably do the same thing.  Or the user might wonder whether or not they do the same thing.  So you may consider hiding one or the other button in that scenario

Reply:
> I thought we decided that the form footer should include an `.nx-btn-bar` and that `.nx-btn-bar` should get this text-align style.

https://github.com/sonatype/sonatype-react-shared-components/pull/122/commits/35beb45c12a1e0bc51400ada141840795012206b